apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/signature/MultiSigData.java

```diff
@@ -260,4 +277,66 @@ public int calcDataSize(int count) {
         }
 
     }
+
+    static class ParserV3 extends Parser{
```

Comment:
v3 parser is not used, i'd suggest to comment it out

Reply:
done